references/optical_flow/train.py

```diff
@@ -310,10 +301,8 @@ def get_args_parser(add_help=True):
     )
     parser.add_argument("--val-dataset", type=str, nargs="+", help="The dataset(s) to use for validation.")
     parser.add_argument("--val-freq", type=int, default=2, help="Validate every X epochs")
-    # TODO: eventually, it might be preferable to support epochs instead of num_steps.
-    # Keeping it this way for now to reproduce results more easily.
-    parser.add_argument("--num-steps", type=int, default=100000, help="The total number of steps (updates) to train.")
-    parser.add_argument("--batch-size", type=int, default=6)
+    parser.add_argument("--epochs", type=int, default=50, help="The total number of epochs to train.")
+    parser.add_argument("--batch-size", type=int, default=2)
```

Comment:
Why the change of the default value of batch-size from 6 to 2?

Reply:
It's unrelated to the PR but all the pretrained weights have relied on bs=2 so I figured I should update it